### DISCUSSION TITLE
Problem regarding Thumbnail in the Search Result field

### DISCUSSION THREAD

**Discussion by raphi-thal** [microsoft-search/pnp-modern-search#1653] [Q&A]
Hallo Everyone,

Short introduction from my side: We are a team of 7 students from a university in austria and we have need to create a IT project in SharePoint.  We got a Product catalog from a company and need to build a solution that every employee of that company can search for the program which he could possibly need in the company doing daily stuff.

We found out that pnp modern search can help us there and create a good search bar/result and filter options.

We have already integrated it but now we came up with a problem. We thought it would be a better user experience if the user gets a familiar picture shown in the result table. Therefore we created a new column and insereted some logos (i have attached a picture). But when we try now to manage the results table we are not able to find the column we can not even create a managed property out of it. What we found out was that there is the possbility to use slots but we were not a 100% sure how that would work in our case. 

It would help us a lot if someone could find a solution to this problem.

Many thanks to all of you 

Raphael and the rest of the Team

<img width="1682" alt="Bildschirmfoto 2021-12-16 um 17 14 13" src="https://user-images.githubusercontent.com/96148866/146408283-7f6ce574-2d4f-49e5-9523-d5685758bfd1.png">
<img width="1602" alt="Bildschirmfoto 2021-12-16 um 17 02 57" src="https://user-images.githubusercontent.com/96148866/146408470-be3df65c-968b-49b4-8e6c-02f6cede0ee4.png">
<img width="931" alt="Bildschirmfoto 2021-12-16 um 17 12 08" src="https://user-images.githubusercontent.com/96148866/146409093-942aa504-01aa-4e48-bfb5-56fff1f549f5.png">
<img width="1385" alt="Bildschirmfoto 2021-12-16 um 17 21 57" src="https://user-images.githubusercontent.com/96148866/146409298-3c2136d1-52b5-479c-89b4-c2c4ee9b71a0.png">

**Comment by wobba** [comment]
Hi, the image column does not emit a crawled property. Reason being the data stored here is a JSON object, not a URL.

You would need to use a text column and store the image URL instead. In the list itself you can use list formatting to render the URL as an image.

Hope this gets you on track.

**Comment by raphi-thal** [comment]
Hi Wobba,

Thx for the help. I think I get it now.
